components/org.wso2.carbon.identity.sso.saml.ui/src/main/resources/org/wso2/carbon/identity/sso/saml/ui/i18n/Resources.properties

```diff
@@ -106,5 +106,6 @@ sp.enter.valid.url=Please enter a valid url
 sp.metadata.valid.file=Please select a valid metadata file
 sp.select.mode=Select Mode
 sp.enable.assertion.query.request.profile=Enable Assertion Query Request Profile
-sp.saml.metadata.certificate.warn=If this SAML service provider metadata file includes a public certificate, please \
-  import it to the Keystore file manually before uploading this file. Do you want to continue?
\ No newline at end of file
+sp.saml.metadata.certificate.warn=Since this the super tenant, Keystore is managed as a file. If this SAML \
+  service provider metadata file includes a public certificate, please import it to the Keystore file manually before \
+  uploading this file. Also, make sure you have replicated it among other nodes as well. Do you want to continue?
```

Comment:
Could reword like following.
You are in super tenant; the Keystore is managed as a file. Please import the public certificate to the Keystore file manually, if this SAML service provider metadata includes a public certificate. Also, make sure to replicate the file across all other nodes. Do you want to continue?